cmd/smith/app/bundle_controller.go

```diff
@@ -122,17 +122,17 @@ func (c *BundleControllerConstructor) New(config *ctrl.Config, cctx *ctrl.Contex
 
 	var catalog *store.Catalog
 	if c.ServiceCatalogSupport {
-		serviceClassInf, err := svcCatClusterInformer(config, cctx, scClient,
+		serviceClassInf, sciErr := svcCatClusterInformer(config, cctx, scClient,
```

Comment:
I think this makes code much less readable. I don't think this is worth "fixing".

Reply:
And in other places too. Not sure what to do - we cannot enable this linter otherwise, right?

Reply:
It's not a big deal if we don't enable this linter

Even upstream doesn't have this enabled by default, because it generates too many false positives: https://golang.org/src/cmd/vet/shadow.go

From the above linked comments:
> A shadowed variable is a variable declared in an inner scope with the same name and type as a variable in an outer scope, and where the outer variable is mentioned after the inner one
is declared.

So, in this particular case, I think it makes sense to extract this `if` out into a separate function, which would separate the scopes and make shadowing a non-issue